### ISSUE TITLE
Jedis Pool Metrics Missing for Some Services

### ISSUE THREAD

**Issue by deverton** [armory-plugins/armory-observability-plugin#35]
The Jedis connection pool metrics show up for Kayenta but not for other services like Fiat.

Fiat:

```
spinnaker@spin-fiat-bb6bcb649-mfldg:/$ wget -q -O - localhost:7003/aop-prometheus | grep -i pool | grep TYPE
spinnaker@spin-fiat-bb6bcb649-mfldg:/$
...

Kayenta

```
bash-5.0$  wget -q -O - localhost:8090/aop-prometheus | grep -i pool | grep TYPE
# TYPE threadpool_maximumPoolSize gauge
# TYPE threadpool_blockingQueueSize gauge
# TYPE threadpool_corePoolSize gauge
# TYPE redis_connectionPool_numWaiters gauge
# TYPE redis_connectionPool_minIdle gauge
# TYPE redis_connectionPool_numActive gauge
# TYPE threadpool_poolSize gauge
# TYPE threadpool_activeCount gauge
# TYPE redis_connectionPool_maxIdle gauge
# TYPE redis_connectionPool_numIdle gauge
```

**Comment by KeisukeYamashita** [comment]
In my Spinnaker, only orca's metrics were missing 👉 https://github.com/spinnaker/spinnaker/issues/6157

**Comment by jasonmcintosh** [comment]
FYI this isn't forgotten and is an issue we have to address - just been not been a priority yet to diagnose unfortunately.... we'd welcome any contributions!

**Comment by massimo-pacher-tw** [comment]
I've stumbled upon the same issue but I've not fixed it yet, cause we're using SQL storage in most of the services, so would need to see how many services are affected.

The class generating those metrics live in [orca](https://github.com/spinnaker/orca/blob/d168010802e6fa06ab24701e97cd28fbd666479e/orca-redis/src/main/java/com/netflix/spinnaker/orca/telemetry/RedisPoolMetricsPostProcessor.java#L46)), and they seem to be pulled in by `kayenta` only ( fiat and gate don't).

That's why you can't see redis pool metrics.

The ideal solution - to keep it DRY - would be to move that class to `kork` telemetry I guess, cause it wouldn't make sense to pull in `orca-redis` on those services just for the sake of an instrumentation class.

Let me know what you think, I'll keep you posted when I manage to work on it.

**Comment by dbyron-sf** [comment]
Or perhaps in [kork-jedis](https://github.com/spinnaker/kork/tree/master/kork-jedis/src/main/java/com/netflix/spinnaker/kork/jedis/telemetry) ?

**Comment by jasonmcintosh** [comment]
Yeah we use SQL almost exclusively so NOT been a HUGE priority... very much appreciate the dig & input!!
